src/services/Authentications.ts

```diff
@@ -0,0 +1,76 @@
+import prisma from "../prisma/client";
+
+const bcrypt = require("bcryptjs");
+
+export default class Authentication {
+  // READ
+  static async findEmail({ email }) {
+    try {
+      return prisma.authentication.findFirstOrThrow({
+        where: {
+          email,
+        },
+      });
+    } catch (e) {
+      throw new Error("something went wrong");
```

Comment:
More explicit error messages perhaps. Only if e contains the error info of course.